### DISCUSSION TITLE
对于延时渲染的滚动列表，鼠标复制文字向下拖动的问题

### DISCUSSION THREAD

**Discussion by hongnny** [idiotWu/smooth-scrollbar#343] [Q&A] [answered]
**对于延时渲染的滚动列表**，当列表滚动到中间或者底部的时候，鼠标拖动复制文字，当鼠标移出可视边界的时候，会出现滚动到底部的问题，代码截图
![image](https://user-images.githubusercontent.com/31148395/65030332-ed765d80-d971-11e9-89ff-dd950a9b4f1e.png)

问题截图
![ta0r9-0wvpi](https://user-images.githubusercontent.com/31148395/65031820-8908cd80-d974-11e9-8fdb-fa6f0ad31ec2.gif)

**Comment by hongnny** [comment]
@idiotWu please help me look at this problem.

**Comment by dolphin-wood** [comment, marked as answer]
Call `scrollbar.update()` after inserting new contents.
